tasks/security-hub.yaml

```diff
@@ -0,0 +1,26 @@
+variables:
+  - name: PKG_PATH
+    description: Path to the zarf package to scan
+  - name: OUTPUT_FILE
+    description: Path to write the results into
+  - name: OUTPUT_FORMAT
+    description: Scan output format
+    default: json
+  - name: VERSION
+    description: Version of uds-security-hub to use
+    default: v0.1.0
```

Comment:
Variables of similar names can easily get mangled - `VERSION` for example may override the default of something else called `VERSION` elsewhere in Maru if the tasks are composed - these should be either refactored into `inputs` (if they are unlikely to change often) or be namespaced with something like `SECURITY_HUB_`.

Reply:
update the variable name